src/inmanta/module.py

```diff
@@ -1657,7 +1664,14 @@ def get_relation_precedence_rules(self) -> List[RelationPrecedenceRule]:
         return [RelationPrecedenceRule.from_string(rule_as_str) for rule_as_str in self.relation_precedence_policy]
 
     def get_index_urls(self) -> List[str]:
-        return [repo.url for repo in self.repo if repo.type == ModuleRepoType.package]
+        # Once setting repos with type package is no longer supported, this method can return self.pip.index_url alone.
+        index_urls_deprecated_option: List[str] = [repo.url for repo in self.repo if repo.type == ModuleRepoType.package]
+        if all([index_urls_deprecated_option, self.pip.index_url]):
+            LOGGER.warning(
+                "Pip indexes are configured in two places. Setting them through the `repo -> url` option has been "
+                "deprecated in favour of the `pip -> index_url` option."
+            )
```

Comment:
I don't really see why we need this log message. The warning about the deprecated option will be logged in the `validate_repo_field()` method, right? With this one we will have two warning messages, with very little added value in my opinion.

Reply:
I agree, I'm in favour of removing this one.
 @sanderr any objections against this ? I seem to recall you asked for a warning if indexes were set in both places but I can't find it anywhere

Reply:
I don't recall, it's possible that was assuming that we would ignore one if the other was set. I agree with your remark.